src/screens/internal/CompareSnapshots/AnomalyCompare.style.ts

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+import { Button, Box } from '@material-ui/core';
+
+export const AnomalyButton = styled(Button)<{ anyNewAnomalies: boolean }>`
+  min-width: fit-content;
+  cursor: pointer;
+  background: ${props => (props.anyNewAnomalies ? 'yellow' : 'white')};
+  font-weight: 500;
+  box-shadow: none;
+  text-transform: none;
+  display: inline-block;
+`;
+
+export const AnomalyModalWrapper = styled(Box)({
+  position: 'absolute' as 'absolute',
```

Comment:
not sure what's going on with `as 'absolute` ?

Also, this seems to be using a different syntax than we usually use.  I'd expect:
```
styled(Box)`
  position: absolute;
  ...
`;

But maybe there's a reason this is different?

Reply:
TBH I don't remember what the `as absolute` was for either, but it works without it 😭 

I was following the syntax/structure in the [`MUI styled()`](https://mui.com/system/styled/#BasicUsage.tsx) docs, but I think that might be using a different engine. I've updated this to match our usual styles